PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalVsPhos.cxx

```diff
@@ -798,7 +926,7 @@ void AliAnalysisTaskEmcalVsPhos::FillClusterHistograms()
   fCaloCells = InputEvent()->GetEMCALCells();
   AliVCaloCells* phosCaloCells = InputEvent()->GetPHOSCells();
   
-  // If MC, get the MC event and utils
+  // If MC, get the MC event
   const AliMCEvent* mcevent;
```

Comment:
maybe this should be set to nullptr just to be sure that something invalid is used?